libafl/src/events/llmp.rs

```diff
@@ -624,7 +627,13 @@ impl<E, S, SP, Z> EventManager<E, Z> for LlmpEventManager<S, SP>
 where
     E: HasObservers<State = S> + Executor<Self, Z>,
     for<'a> E::Observers: Deserialize<'a>,
-    S: UsesInput + HasExecutions + HasClientPerfMonitor + HasMetadata,
+    S: UsesInput
```

Comment:
you don't need these type constraints

Reply:
<img width="939" alt="image" src="https://github.com/AFLplusplus/LibAFL/assets/14891443/134274f0-d7d1-4d7f-9883-cc0b649d2890">
There is an error.

Reply:
hm but why? can you give me full error log?

Reply:
because here you don't even use those methods then why is it necessary?

Reply:
![Uploading image.png…]()

Reply:
<img width="960" alt="image" src="https://github.com/AFLplusplus/LibAFL/assets/14891443/392479d9-a7db-4c30-9639-210def4da224">

Reply:
but you don't need this

Reply:
I mean you could just store imported, pending in state, and don't make any traits.
then you don't need these type constraints anymore

Reply:
ok